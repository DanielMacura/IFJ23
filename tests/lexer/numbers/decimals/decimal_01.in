let x = 3.14
var y = 42
let z = 2.71828