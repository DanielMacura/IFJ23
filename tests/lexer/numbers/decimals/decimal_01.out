KEYWORD
IDENTIFIER
EQUALS
FLOAT:3.14
KEYWORD
IDENTIFIER
EQUALS
INTEGER:42
KEYWORD
IDENTIFIER
EQUALS
FLOAT:2.71828
EOF
